src/probnum/__init__.py

```diff
@@ -34,18 +34,23 @@
     randvars,
     utils,
 )
+from ._function import Function, LambdaFunction
 from ._version import version as __version__
 from .randvars import asrandvar
 
 # Public classes and functions. Order is reflected in documentation.
 __all__ = [
     "asrandvar",
+    "Function",
+    "LambdaFunction",
```

Comment:
```suggestion
```
I'm not actually sure that we want those here, since `from probnum import *` would import `Function` into the current namespace, but I guess that's the user's fault then 😁

Reply:
I guess it's the same with PN method components. Eventually we could move this to `backend` maybe? For now if we want it to be documented than it should be imported into the `probnum` namespace if the module is there.